include/unifex/inplace_stop_token.hpp

```diff
@@ -98,11 +100,17 @@ class inplace_stop_token {
 
   inplace_stop_token() noexcept : source_(nullptr) {}
 
+  inplace_stop_token(const inplace_stop_token& other) noexcept = default;
```

Comment:
Why make it copyable?

Reply:
The stop token concept should allow copyability.
It should have handle/pointer semantics to whatever the underlying state is.
`std::stop_token` is copyable.